.github/labeler.yml

```diff
@@ -0,0 +1,47 @@
+"L: ruby:bundler":
+  - bundler
```

Comment:
All the examples in the [actions/labeler README](https://github.com/actions/labeler#basic-examples) include wildcard matchers. This is my first encounter with this Action. Do we need to use `bundler/*`?

Reply:
You're right, it's my first encounter with this action too!

I don't think this is going to work as is, but maybe it does and the labeler README needs better docs?

I'll do some tests to figure it out.

Reply:
I wasn't sure how to test this without merging it. And I'm pretty sure the folder only approach was not going to work, so I changed this to use a proper glob.

Reply:
For future references, you can always test this(and any other automation) using a fork. That'd be less destructible in case there is any problem as you can recreate the fork anytime

Reply:
Thanks for the tip. I tried it on my fork and it worked as expected!